README.adoc

```diff
@@ -10,7 +10,7 @@
 :icons: font
 :toc: left
 
-image:https://travis-ci.org/arquillian/arquillian-cube.svg?branch=master["Build Status", link="https://travis-ci.org/arquillian/arquillian-cube"]
+image:https://travis-ci.org/arquillian/arquillian-cube.svg?branch=main["Build Status", link="https://travis-ci.org/arquillian/arquillian-cube"]
```

Comment:
Well, you can remove this line altogether - it doesn't resolve.

Reply:
Thanks, fixing.